deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1243,6 +1243,14 @@ end}.
 {mapping, "collect_statistics_interval", "rabbit.collect_statistics_interval",
     [{datatype, integer}]}.
 
+%% Enable/disable file_handle_cache statistics. Disabeling this
+%% can help in cases of lock contention on the backing ets table.
+%%
+%% {collect_file_handle_cache_statistics, true},
+
+{mapping, "collect_file_handle_cache_statistics", "rabbit.collect_file_handle_cache_statistics",
```

Comment:
I'd much prefer `file_handles.collect_statistics` or something like that. `collect_statistics_interval` was a name chosen in 2010 or so.